cpp/CMakeLists.txt

```diff
@@ -267,8 +267,6 @@ if(BUILD_CUML_CPP_LIBRARY)
         src/explainer/kernel_shap.cu
         src/explainer/permutation_shap.cu
         src/explainer/tree_shap.cu
-        src/fil/fil.cu
-        src/fil/infer.cu
```

Comment:
@dantegd asked to deduplicate the compilation targets. As I've been having discrepancies between pure cmake and build-script results, I'd prefer to address this part sooner rather than later.

Reply:
This would be better in a separate pull request.